dhall/src/Dhall.hs

```diff
@@ -2527,13 +2533,6 @@ injectProject =
 
 -}
 
--- | Infix 'divided'
-(>*<) :: Divisible f => f a -> f b -> f (a, b)
-(>*<) = divided
-
-infixr 5 >*<
-
--- | Intermediate type used for building a `ToDhall` instance for a record
```

Comment:
Why did you remove this line of doc?

Reply:
Oh sorry, I understand what you did. I didn't notice that above `(>*<)` there was the doc for `RecordEncoder`.